python_modules/libraries/dagster-dg/dagster_dg/utils/__init__.py

```diff
@@ -31,6 +32,27 @@
 CLI_CONFIG_KEY = "config"
 
 
+def is_windows() -> bool:
+    return platform.system() == "Windows"
+
+
+def is_macos() -> bool:
+    return platform.system() == "Darwin"
```

Comment:
high confidence this covers all mac flavors?

Reply:
Probably does, but changed to `sys.platform == "darwin"` since that is likely most robust.